brainframe/cli/commands/utils.py

```diff
@@ -23,3 +26,15 @@ def subcommand_parse_args(parser: ArgumentParser):
         args.noninteractive = True
 
     return args
+
+
+def requires_root(function):
+    """A decorator that checks if the user is root before running a function"""
+
+    def wrapper(*args, **kwargs) -> Any:
+        if not os_utils.is_root():
```

Comment:
I'd recommend using [`functools.wraps`](https://docs.python.org/3/library/functools.html#functools.wraps)

Reply:
Also, I saw you delete the type hints, but you may as well annotate that `function` is a `callable`

Reply:
Good idea